common/src/main/java/com/revenuecat/purchases/common/networking/Endpoint.kt

```diff
@@ -33,6 +33,22 @@ sealed class Endpoint(val pathTemplate: String, val name: String) {
     }
 
     val supportsSignatureValidation: Boolean
```

Comment:
Also this isn't used now?

Reply:
Done in iOS: https://github.com/RevenueCat/purchases-ios/pull/2752

Reply:
> Also this isn't used now?

It's used in the signing manager: https://github.com/RevenueCat/purchases-android/blob/ef34062996a17aa6a47d6eb2ca0903d66a51ab5e/purchases/src/main/kotlin/com/revenuecat/purchases/common/verification/SigningManager.kt#L23